portfolio/src/main/webapp/script.js

```diff
@@ -91,3 +91,8 @@ function changeImageBorders(color){
     document.images.item(i).style.border = "10px solid " + color;
   }
 }
+
+function sayHi() {
+  fetch('/data').then(response => response.text()).then(
+    document.getElementById('greeting-container').innerText = 'Hello Meagan!');
```

Comment:
Did you mean to use response.text() here?

Reply:
Also, this doesn't look syntactically correct... you probably meant `() =>` or `(text) =>`

Reply:
I saw it in the example code, is it not necessary here? I wasn't sure what Promises had to be satisfied before I sent the greeting.